docs/installation.rst

```diff
@@ -96,6 +96,13 @@ Install Molecule:
 
     $ python3 -m pip install --user "molecule[lint]"
 
+Molecule does not include ansible-lint (nor does the lint extra), but
+is easily installed separately:
+
+.. code-block:: bash
+
+    $ python3 -m pip install --user "molecule[lint] ansible-lint"
```

Comment:
Remove lint extra from here, it is used only internally.

Reply:
Ahh... maybe this is where all of the confusion originally stems from? The [current documentation](https://molecule.readthedocs.io/en/latest/installation.html) instructs users to install via:
```bash
$ python3 -m pip install --user "molecule[lint]"
```

If I understand correctly, `molecule[lint]` shouldn't be referenced in the user documentation at all. Is this correct? If so, I'll double check that there are no other references to it in the user docs and fix them all in this PR.

Reply:
You are right, that's mostly for internal use. There are no direct imports from molecule itself to any stuff inside lint exta.